manifest/marshal.go

```diff
@@ -21,3 +21,12 @@ func marshalRequest(obj *types.ManifestRequest) ([]byte, error) {
 	}
 	return buf.Bytes(), nil
 }
+
+func marshal(obj *types.Manifest) ([]byte, error) {
```

Comment:
obj can be github.com/ovrclk/akash/vendor/github.com/gogo/protobuf/proto.Message